src/main/java/frc/robot/Ports.java

```diff
@@ -1,5 +1,9 @@
 package frc.robot;
 
 public final class Ports {
-
+public class Intake{
+    public static final int HIGHER_SPARK_MAX_PORT = 0;
+    public static final int LOWER_SPARK_MAX_PORT = 0;
```

Comment:
"motor" is enough, no need to specify the motor controller type.

Reply:
done.